src/settings/general-settings.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Pane } from '@folio/stripes/components';
+
+export default class GeneralSettings extends React.Component {
+  static propTypes = {
+    label: PropTypes.object.isRequired,
+  };
+
+  render() {
+    return (
+      <Pane defaultWidth="fill" fluidContentWidth paneTitle={this.props.label}>
```

Comment:
It is a good approach to place each prop on the new line with callback props at the bottom.

Reply:
Fixed